### PR TITLE
[IMPROVEMENT] Support for 'twitter://' scheme internal browser

### PR DESCRIPTION
**Pull request by andreahaku**
### **Description**
Attempting to reliably link back to the Twitter mobile app from MetaMask and is not able to do this on iOS.

Confirmed flow: 
User selected Choose NFT -> Deep link example: https://metamask.app.link/dapp/https://twitter.com/ -> Dapp browser and triggers App Request for Wallet Connections -> On success -> twitter:// called -> return to Twitter

### **Technical Details**
-   To support this link back functionality `twitter://` needs to be added the URL scheme

### **Acceptance Criteria**
-   Install Twitter on iOS test device
-   Create a link that can be loaded on the browser `twitter://test`
-   The link above should be open the Twitter app

### **References**
-   https://medium.com/wolox/ios-deep-linking-url-scheme-vs-universal-links-50abd3802f97

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

File: app/core/DeeplinkManager.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ class DeeplinkManager {
 	}
 
 	parse(url, { browserCallBack, origin, onHandled }) {
-		const urlObj = new URL(url);
+		const urlObj = new URL(url.replace('http://', '').replace('http://', ''));
```
Comment (rickycodes) [inline app/core/DeeplinkManager.js:95]: same replace call twice?

**Review by rickycodes — CHANGES_REQUESTED** [review]
noticed one small thing

File: app/core/DeeplinkManager.js (hunk 1 above)
Comment (andreahaku) [reply app/core/DeeplinkManager.js]: You’re right :) thanks a lot!

**Review by cortisiko — CHANGES_REQUESTED** [review]
@andreahaku  sending this back your way. Deep links appears to be broken.

to reproduce:
attempt to deep link using: https://metamask.app.link/dapp/uniswap.eth


see android: http://recordit.co/uWHM733AZ5 
see iOS: http://recordit.co/f0ge8PFe2S 


In terms of testing what this actually does, I attempt to deep link using  twitter://test  and it successfully launches the twitter app. Should I do anything extra or the twitter app launching is sufficient?

**Comment by andreahaku** [comment]
> @andreahaku sending this back your way. Deep links appears to be broken.
> 
> to reproduce: attempt to deep link using: https://metamask.app.link/dapp/uniswap.eth
> 
> see android: http://recordit.co/uWHM733AZ5 see iOS: http://recordit.co/f0ge8PFe2S
> 
> In terms of testing what this actually does, I attempt to deep link using twitter://test and it successfully launches the twitter app. Should I do anything extra or the twitter app launching is sufficient?

Deeplinks should be fixed now. Yes it should open Twitter when using "twitter://" scheme from the browser and you can also test it using the deeplink that is the description. Thanks :)

**Review by cortisiko — APPROVED** [review]
This is QA passed!

**Review by rickycodes — APPROVED** [review]

**Pull request merged**